scripts/classes/settings.mjs

```diff
@@ -0,0 +1,70 @@
+import { ItemsWithSpells5e } from "../items-with-spells-5e";
+
+export const EXCLUDED_TYPES = [
+  "class",
+  "subclass",
+  "background",
+  "race",
+  "spell",
```

Comment:
if spells can be attached to spells ever, you have good odds of causing bad things to happen, this one i think shouldn't be possible

Reply:
This is the list of "absolutely never" item types.